javascript-view-swapping/main.js

```diff
@@ -0,0 +1,27 @@
+var $tabs = document.querySelectorAll('.tab');
+var $views = document.querySelectorAll('.view');
+
+var $tabContainer = document.querySelector('.tab-container');
+$tabContainer.addEventListener('click', tabHandler);
+
+function tabHandler(event) {
+  if (event.target.matches('.tab')) {
```

Comment:
This would be a great time to use a guard. Rather than only doing things if the target has the `tab` class, you should check at the beginning if the element doesn't have a `tab` class, and if it doesn't, `return` out of the function.